src/lib/form/useCreateModel.ts

```diff
@@ -13,9 +13,9 @@ export const useCreateModel = (resource: string) => {
                     type: 'create',
                     data: data,
                 })
-                return { response }
+                return { data: response }
             } catch (error) {
-                return { [FORM_ERROR]: (error as Error | string).toString() }
+                return { error: parseErrorResponse(error) }
```

Comment:
I removed the `[FORM_ERROR]` here, because I don't think it's correct to leak "form-related" stuff into the API layer.

However, I'm not totally sure if we should parse the error here (as we do that in `createFormError` as well (if needed).